models/types.ts

```diff
@@ -133,17 +134,22 @@ export enum Insurance {
 }
 
 export enum Accessibility {
-    "Speech",
-    "Vision",
-    "Hearing",
-    "Mental",
-    "Digital Literacy"
+    Speech = "Speech",
+    Vision = "Vision",
+    Hearing = "Hearing",
+    Mental = "Mental",
+    DigitalLiteracy = "Digital Literacy"
 }
 
-export interface Resource extends DocumentData {
-    id: string;
-    name: string;
-    description: string;
-    incomeLevel: number;
-    employed: boolean
+export enum Language {
+    English = "English",
+    Spanish = "Spanish"
+}
+
+export enum ResourceCategory {
+    Childcare = "Childcare", 
+    Healthcare = "Healthcare", 
+    FinancialHelp = "Financial Help", 
+    ImmigrationAssistance = "Immigration Assistance", 
+    Housing = "Housing"
 }
```

Comment:
So small note on this, although Childcare = "Childcare" doesn't look as clean here, it actually makes it easier to retrieve a list of strings for an enum. Without it, when you do Object.keys(ResourceCategory), you get a list that looks like ['Childcare', 'Healthcare',...'1','2','3'...] because it assumes that numbers are the keys associated with an enum. Adding the Childcare = "Childcare" to the enum makes it to a call to Object.keys(ResourceCategory) only returns a list of resource categories. 

Just JS things